Assets/Scripts/Enemy.cs

```diff
@@ -323,5 +367,75 @@ private void Charge()
         }
     }
 
+    //multiple patterns of boss monster
+    private IEnumerator Patterns()

```

Comment:
This would be a good method to use a summary comment (three slashes above the method name) ///